TranslateTrajectoryPayload payload
---
float64[3] result